bpf/lib/policy.h

```diff
@@ -15,9 +15,11 @@ static __always_inline int
 __account_and_check(struct __ctx_buff *ctx, struct policy_entry *policy,
```

Comment:
I see the report from @LeiW000 (thanks for testing!), please mark ctx as `__maybe_unused`.
```suggestion
__account_and_check(struct __ctx_buff *ctx __maybe_unused, struct policy_entry *policy,
```

Reply:
@LeiW000 @joestringer thanks, I've made changes based on those reviews.